css/main.css

```diff
@@ -145,68 +149,77 @@ kbd {
 	display: block;
 }
 
+.flexTopBar {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	top: 0;
+	left: 0;
+	z-index: 1;/*Lets topbar render ontop of iframe*/
+	user-select: none;
+}
+
+.spectateBanner::before{
+	content: "👁️ ";
+	font-style: normal;
+}
 .spectateBanner{
 	background-color: #90FF90;
 	cursor: pointer;
-	text-align:center;
+	flex: 0 0 35%;
+	width: 99.5%; /* I think this is due to a padding issue or something? */
+	text-align: center;
+	font-style: italic;
+	background-color: lightgreen;
+	border-style: outset;
+	border-color: lightgreen;
+	font-size: 1em;
```

Comment:
1 em is default i think?

Reply:
This is a bad habit of mine where I originally thought this would override any inherited font size from parent elements or something to make it the default. I removed several instances of this from main.css, none of which caused anything to change when removed. Also somehow in the conflict resolution phase I ended up with a complete duplicate of the warning class in the css file so I removed the duplicate.